scripts/OIMS2fits.py

```diff
@@ -181,30 +183,39 @@ def main(args):
                         print("    integration time: %.3f s" % tInt)
                         print("    frequency: %.3f MHz" % header['freq'])
                     
+                    if station==b'LWASV':
+                        # SV has special correction factors to improve positionsQ
+                        w = getSVwcs(hdr, imSize)
+
+                    else:
+                        w = getGENERICwcs(hdr, imSize)
+                    headertmp = w.to_header()
```

Comment:
Do we also need RADESYS and EQUINOX like in [lsl.imaging.utils.ImgWPlus.wcs](https://github.com/lwa-project/lsl/blob/main/lsl/imaging/utils.py#L1546)?

Reply:
I do not know about these options. Not sure if necessary.

Reply:
I guess the test failure in [4c4e213](https://github.com/lwa-project/orville_wideband_imager/pull/29/commits/4c4e2138dbe856a347426941955005927534e28b) suggests that RADESYS does matter.